webview/src/plots/components/Plots.tsx

```diff
@@ -35,7 +35,9 @@ const PlotsContent = () => {
     const onResize = () => {
       wrapperRef.current &&
         dispatch(
-          setSnapPoints(wrapperRef.current.getBoundingClientRect().width - 100)
+          setMaxNbPlotsPerRow(
+            wrapperRef.current.getBoundingClientRect().width - 100
```

Comment:
[Q] Can this pass `wrapperRef.current.getBoundingClientRect().width` only and do the full calc in the slice? Having `-100` here confused my tiny brain. All the logic would then be in a single place too.

Reply:
I was thinking the opposite. Having `-100` so far away from the actual item we're calculating the width from is going to cause more confusion. That said, `100` probably refers to something (margin or padding maybe?), I'll label it correctly, because even I can't remember where it comes from.